### PR TITLE
feat(config): add config file for command builds and readme

### PR DESCRIPTION
**Pull request by andreas-stegmaier-fnt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+To find out what the rules are about and what rules can be used see:
+
+https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+
+# Setup commitlint for local testing
+
+## Install
+
+``` bash
+# Create a new directory and navigate into it
```
Comment (ghost) [inline README.md]: In theory we could also provide a developer setup within this repository... Then you would only have to run something like `npm ci` and would be ready to go 😄

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## Commitlint config file
```
Comment (ghost) [inline README.md]:
```suggestion
# Commitlint config file
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+For checking the commit messages, a configuration file (`commitlint.config.js`) with some rules is used, which can be individually adapted.
+To find out what the rules are about and what rules can be used see:
+
+https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+
+# Setup commitlint for local testing
```
Comment (ghost) [inline README.md]:
```suggestion
## Setup commitlint for local testing
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## Install
```
Comment (ghost) [inline README.md]:
```suggestion
### Install
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+# install npm dependencies
+npm install --save-dev @commitlint/cli
+npm install --save-dev @commitlint/config-conventional
+
+# check if installation was successfull (help message should be shown)
+npx commitlint
+```
+
+## Commitlint definition file
```
Comment (ghost) [inline README.md]:
```suggestion
### Commitlint definition file
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    'scope-empty': [2, 'never'],
+    'scope-case': [2, 'always', 'lower-case'],
+    'subject-empty': [2, 'never'],
+	'body-leading-blank': [2, 'always'],
+    'body-max-line-length': [2, 'always', 100]
+  },
+};
+```
+
+## Testing
```
Comment (ghost) [inline README.md]:
```suggestion
### Testing
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
+## Commitlint config file
+For checking the commit messages, a configuration file (`commitlint.config.js`) with some rules is used, which can be individually adapted.
+To find out what the rules are about and what rules can be used see:
+
+https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+
```
Comment (ghost) [inline README.md:6]: I would also love to have an example here how commitlint can be used in GitHub Actions workflows

**Comment by ghost** [comment]
Please also add a CODEOWNERS file

File: README.md (hunk 1 above)
Comment (andreas-stegmaier-fnt) [reply README.md]: i will contact you on monday :-)

File: README.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+### Testing
+```bash
+echo "feat(scope): new feature" | npx commitlint
+```
+
+alternatively you can read multiline input from a file:
+```bash
+cat input.txt | npx commitlint
```
Comment (ghost) [inline README.md]:
```suggestion
cat input.txt | npx commitlint --config cmd-commitlint.config.js
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+```
+or check actual commit message and using a separate config file stored at different location:
+```bash
+npx commitlint --from=HEAD~1 --config C:/Temp/commitlint/my-commitlint.config.js
```
Comment (ghost) [inline README.md]:
```suggestion
npx commitlint --from=HEAD~1 --config cmd-commitlint.config.js
```

**Comment by ghost** [comment]
Everything else looks fine for me 😄

**Review by ghost — APPROVED** [review]

**Pull request merged**